getfastq.sh

```diff
@@ -102,41 +103,63 @@ function _progress_getfastq {
         printf "Bad TARGETS path '$1'.\n"
         exit 1 # Argument failure exit status: bad target path
     fi
-
-    local log_file="$(find "${target_dir}" \
-        -maxdepth 1 -type f -iname "Z_getFASTQ_*.log")"
-
-    if [[ -n "$log_file" ]]; then
-
-        # NOTE: the -- is used here to indicate the end of 'bash' command
-        #       options and the beginning of Bash script arguments.
-        # NOTE: possible non-matching greps inside 'find ... -exec bash' won't
-        #       break the script, even when 'set -e' is active.
-        printf "\n${grn}Completed:${end}\n"
-        find "${target_dir}" -maxdepth 1 -type f -iname "Z_getFASTQ_*.log" \
-            -exec bash -c '
-                grep -E " saved \[| already there;" "$1"
-            ' -- {} \;
-
-        printf "\n${red}Failed:${end}\n"
-        find "${target_dir}" -maxdepth 1 -type f -iname "Z_getFASTQ_*.log" \
-            -exec bash -c '
-                grep -E ".+Terminated| unable to |Not Found." "$1"
-            ' -- {} \;
-
-        printf "\n${yel}Incoming:${end}\n"
-        find "${target_dir}" -maxdepth 1 -type f -iname "Z_getFASTQ_*.log" \
-            -exec bash -c '
-                dead_track=$(tail -n 3 "$1" | grep -E \
-                    " saved \[| already there;|Terminated|unable to|Not Found")
-                [[ -z $dead_track ]] && tail -n 1 "$1" && echo
-            ' -- {} \;
-        exit 0 # Success exit status
-    else
+    
+    declare -a logs=()
+    readarray -t logs < <(find "${target_dir}" -maxdepth 1 -type f -iname "Z_getFASTQ_*.log")
```

Comment:
Elegant, I like it! ...but it fails reporting the status of all files when the sequential mode is used, since in that case a single log file is generated with multiple download report inside...

Reply:
Oh, I think multiple logs would be generated... Maybe we could change it to producing a different log per spawned worker? It might even be clearer...

Reply:
It would be the best choice!!
But I cannot say right now what pain would it take... in the sequential mode there is only one `nohup bash ... ` instance, whose single log would need to be split into many files (one per FASTQ). Or we should completely rethink that statement.

Reply:
Fixed together with above.